ADAL/src/ui/ADWebAuthController.m

```diff
@@ -250,15 +251,31 @@ - (void)webAuthDidStartLoad:(NSURL*)url
 
 - (void)webAuthDidFinishLoad:(NSURL*)url
 {
-    AD_LOG_VERBOSE(_requestParams.correlationId, @"-webAuthDidFinishLoad host: %@", url.host);
+    if ([ADAuthorityUtils isKnownHost:url])
```

Comment:
I'd use a ternary and print both non-pii and pii.
Also, if block is missing logging for pii.